visualization_scripts/README.md

```diff
@@ -0,0 +1,26 @@
+## Visualization scripts
+**Requirements**
+
+Cytoscape v3.7.1
+
+Anaconda 
+
+**Setup**
+
+Before running the visualization script, edit the run.py file and within the main function. Edit `DIRNAME` and insert the directory that Cytoscape can be found at on your machine. If you do not know where `Cytoscape.exe` is located, you can choose to search the highest directory availible. The script is set to traverse you machines file system starting in the Users directory. 
+
+**Run the Script**
+
+From the `visualization_scripts` working direstory of tps call:
+
+`bash activate.sh output.sif "\tps\visualization_scripts\style.xml"`
```

Comment:
This raises interesting usage questions.  What is the starting point of the user's system when they want to run this script?  Do we want to automatically install and activate the environment for them?  That would require different scripts for different operating systems.  Alternatively, we could require the user to already have the conda environment available and activated.  That puts a bigger burden on the user instead of our scripts.

Reply:
As of right now, I have been doing all of my implementation from the visualization_scripts folder in my experimental branch of tps called visualization. The only real requirement for the scripts to run is to have an environment that is running python 3.5 with py2cytoscape installed.

Reply:
I believe `"\tps\visualization_scripts\style.xml"` should be `style.xml` because the script is run from the `visualization_scripts` directory.